config/staging.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
we likely want to hide these as well, and not push them to github otherwise anyone could start pushing to our firebase account